data/themes.yml

```diff
@@ -255,3 +255,4 @@
 - { colors: '#323232,#E60A00,#E60A00,#FFFFFF,#505050,#F5F5F5,#E64600,#E60A00', name: '360Channel' }
 - { colors: '#000639,#3E313C,#108043,#FFFFFF,#173630,#FFFFFF,#50B83C,#9C6ADE', name: 'Shopify' }
 - { colors: '#100E0E,#C10800,#FFB400,#FFFFFF,#242121,#FFFFFF,#FF2B22,#FF2B22', name: 'RubyConf' }
+- { colors: '#1D1F2B,#2C3849,#ACE2CC,#1D1F2B,#41455C,#FFFFFF,#C9E570,#FE8394', name: 'Monokai Pro' }
```

Comment:
Shouldn't this be more like:
```
#2C292D,#383933,#FC9867,#FDF9F3,#AB9DF2,#FDF9F3,#A9DC76,#FF6188,#2C292D,#FDF9F3
```

🤔

Reply:
I just saw this, LOL ...So yea, that's the default Monokai Pro version, I did the colors for the Monokai Pro Octagon